### PR TITLE
 Standardize test hetnets & cache URL reads

### PR DESCRIPTION
**Pull request by dhimmel**
Ready for review

### CODE REVIEW

File: hetmech/tests/hetnets.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import hetio.readwrite
+
+import hetmech.hetmat
+
+format_github_url = 'https://github.com/{repo_slug}/raw/{commit}/{path}'.format
+
+hetnet_urls = {
+    # Figure 2D of Himmelstein & Baranzini
+    # (2015) PLOS Comp Bio. https://doi.org/10.1371/journal.pcbi.1004259.g002
+    'disease-gene-example':
```
Comment (zietzm) [inline hetmech/tests/hetnets.py]: Nice. I like the simplification from having url on every test

File: hetmech/tests/hetnets.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    """
+    If hetmat=True, import graph into a hetmat located on-disk at directory.
+    """
+    if name not in hetnet_urls:
+        raise ValueError(
+            f'{name} is not a supported test hetnet.\n'
+            'Choose from the following currently defined hetnets: ' +
+            ', '.join(hetnet_urls)
+        )
+    if name not in hetnet_io_cache:
```
Comment (zietzm) [inline hetmech/tests/hetnets.py]: In other files you import just this function. Does the hetnet_io_cache, defined in this file, get transferred to others?

Comment (dhimmel) [reply hetmech/tests/hetnets.py]: When you import a function, it retains the scope of its source file.

**Comment by dhimmel** [comment]
@zietzm I added a test of our cache in https://github.com/greenelab/hetmech/pull/121/commits/6839437f1d669d63e9ef7d84920b6f6b7eabcfcf

**Pull request merged**